data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -906,7 +906,7 @@
 
                 [message]
                     speaker=Elenia
-                    message= _ "Yes, I would... like to join... you Lord Tallin."
+                    message= _ "Yes, I would... like to join you... Lord Tallin."
```

Comment:
How does she know Tallin's name? I read the ellipsis as indicating that she's recently recovered consciousness; so neither the old nor new versions seem fitting. Either suggest that she's been refreshed and had a chat while Sister Thera was healing her, or give her less information.

Reply:
That should fix it. 

I made when the unit finds her they shout for Tallin.

Reply:
Still doesn't feel good to me, given that that's when she's even less conscious. Also, there's a capital `L` that shouldn't be there in the new line.

Reply:
Still doesn't feel good to me, given that that's when she's even less conscious. Also, there's a capital `L` that shouldn't be there in the new line.

Reply:
I think it's fine. I did add the ... back.

The issue is I have no idea where to Tallin's it without rewriting the entire scene, and I don't think we can remove it.

Also, I thought she'd be sort of conscious, enough to hear it.